src/index.ts

```diff
@@ -913,6 +934,60 @@ class ThreadPool {
 
     await Promise.all(exitEvents);
   }
+
+  async close (options : CloseOptions = kDefaultCloseOptions) {
+    this.closingUp = true;
+
+    if (options.force) {
+      const skipQueueLength = this.skipQueue.length;
+      for (let i = 0; i < skipQueueLength; i++) {
+        const taskInfo : TaskInfo = this.skipQueue.shift() as TaskInfo;
+        if (taskInfo.workerInfo === null) {
+          taskInfo.done(new AbortError());
+        } else {
+          this.skipQueue.push(taskInfo);
+        }
+      }
+
+      const taskQueueLength = this.taskQueue.size;
+      for (let i = 0; i < taskQueueLength; i++) {
```

Comment:
The equivalent will be:

```suggestion
	for (const taskInfo of skipQueue) {
```